Example/openxcframework/SendCanViewController.swift

```diff
@@ -39,7 +38,7 @@ class SendCanViewController: UIViewController, UITextFieldDelegate {
         bm = BluetoothManager.sharedInstance
         
         // set default CAN target
-        vm.setCanDefaultTarget(self, action: SendCanViewController.default_sendcan_rsp)
+        vm.setCanDefaultTarget(self, action: SendCanViewController.defaultSendcanResponse)
```

Comment:
The variable name should be defaultSendCanResponse

Reply:
done but this is function name .